MobileFilter/sections/specific_web.txt

```diff
@@ -4257,6 +4257,13 @@ popularmechanics.com#?#section > div[class*=" "]:has(> div[class]:first-child +
 veranda.com,wetter.com#?#div[class^="sc-"]:has(> div.ad-container)
 wetter.com#?#.som-wrapper:has(> div[data-slotname])
 wetter.com#?#.clickperformance-wrapper:has(> .ad-flex)
+wetter.com#?#.mbanner-fixed:remove()
+wetter.com#$#.mbanner-is-fixed { top: 0px !important; padding-top: 0px !important; }
+wetter.com#$##main-header-fixable { top: 0px !important; }
+wetter.com#?#div.ad-container:remove()
```

Comment:
```suggestion
wetter.com#?#div.ad-container:remove()
```

It is not visible on my end (on the main page)

<details>
<summary>Details</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/2874fc88-fe88-4839-aae7-82508f36c725)


</details>

Reply:
It's there, but I see now it's already being hidden by:
```
MobileFilter/sections/specific_web.txt:1601:wetter.com##body div.ad-container
```
, so my rule is superfluous. I forgot to enable the Mobile filter at first when doing this task :harold: